source/reference/minio-mc/mc-license-register.rst

```diff
@@ -11,6 +11,10 @@
 .. mc:: mc support register
 .. mc:: mc license register
 
+.. important:: 
+
+   ``mc license register`` requires :ref:`MinIO Client <minio-client>` version ``RELEASE.2023-11-20T16-30-59Z`` or newer.
+
 Command History
```

Comment:
The `Command History` was useful earlier for customers to know which command they can use with their version of `mc`. 

However, now that the older versions of `mc` will not work at all for registration, it might be better to remove this section altogether.

Reply:
@anjalshireesh does that mean, in general, that users *must* update their `minio` server before registration?
Since generally we want `mc` and `minio` to be closely aligned.

What versions of `mc` are effectively no longer compatible with SUBNET?

Reply:
That makes sense. Removing.

Reply:
@ravindk89 

> does that mean, in general, that users must update their minio server before registration?
Since generally we want mc and minio to be closely aligned.

No, only `mc` needs to be upgraded. While somehow the message that `mc` and `minio` should be closely aligned has got popular, it is not correct and ideally latest `mc` should always maintain compatibility with older `minio` versions. cc @abperiasamy 

Also,  there are other alternatives if upgrading mc is not possible for some important (customer-specific) reason, like using console for registration or using a lower level mc command.

> What versions of mc are effectively no longer compatible with SUBNET?

Any version lower than `RELEASE.2023-11-20T16-30-59Z` cannot be used for `mc license register` or `mc license update`. For already-registered clusters, older mc versions will still work for the `mc support` commands.